### ISSUE TITLE
Bug: Events with indexed fields do not show up

### ISSUE THREAD

**Issue by Thegaram** [ObsidianLabs/ConfluxStudio#23]
Events with indexed fields do not show up under the Events tab on the Contracts screen. To reproduce, try this contract:

```solidity
pragma solidity >=0.5.0 <0.7.0;

contract Contract {
    event Event(uint256 indexed num);

    constructor() public {
        emit Event(0);
    }
}
```

**Comment by qftgtr** [comment]
It can show up now. If there are more issues please reopen it.

**Comment by Thegaram** [comment]
I believe this is still an issue in the latest release @qftgtr.

**Comment by Thegaram** [comment]
Still not fixed in v0.8.0-rc.1.

```solidity
pragma solidity ^0.8.0;

contract TestToken {
    event MyEvent(uint256);
    event MyEvent2(uint256 indexed);

    function bar() public {
        emit MyEvent(0);  // <-- this shows up
        emit MyEvent2(0); // <-- this does not
    }
}
```

**Comment by qftgtr** [comment]
This issue should be fixed since v0.8.0-rc.3
